initfiles/etc/DIR_NAME/environment.conf.in

```diff
@@ -28,8 +28,9 @@ mpStart=7101
 mpEnd=7500
 mpSoMaxConn=128
 mpTraceLevel=0
-#enable SSL for dafilesrv remote file access
-#dfsUseSSL=false
+# enable SSL for dafilesrv remote file access (SSLNone|false / SSLBoth|true / SSLOnly)
+dfsUseSSL=SSLNone
+#dfsSSLPassPhrase=
```

Comment:
Currently we don't provide a standalone method to create the MD5 encrypted passphrase.  But I added this as a first step in the process of getting these into configmgr properly.
It still works as before, you can generate a key file without a passphrase.